### ISSUE TITLE
option to disable default loader / voter / guesser

### ISSUE THREAD

**Issue by dbu** [symfony-cmf/SeoBundle#242]
the default things should be possible to disable. not sure how to not make this horribly verbose... ideas? just a global on/off?

**Comment by ElectricMaxxx** [comment]
Doesn't WE have a priority to overwrite them? This would he the first solution. 
The second would be to create a enable/disable map which can be passed to the voters. They can decline when their own option is set to false. Same for guesser, they can decline in `supports()` method.

**Comment by dbu** [comment]
for guessers yes, only if i want some value not at all, there is still a problem. for the loader, its more important because i might have phpcr enabled but need my own, more clever, loader instead of the default one...

maybe a section in the defaults? 
```yaml
cmf_seo:
    sitemap:
        defaults:
            loaders: none|all|phpcr
            guessers: none|all|<comma separated list>
            voters: none|all|publish-workflow            
```

then in the DI extension we just remove the services from the container that are not activated.

**Comment by ElectricMaxxx** [comment]
Or just remove the tag?

**Comment by dbu** [comment]
why would we keep the services lying around? we actually should even
make them private, no point in other things accessing them.

**Comment by ElectricMaxxx** [comment]
:+1:

**Comment by ElectricMaxxx** [comment]
`none` would be default for all or `all`?

**Comment by ElectricMaxxx** [comment]
seems wrong and hard to implement to have a different usage on them, so i would suggest to use `none|all|<comma-sepparated-list>` for all.

**Comment by ElectricMaxxx** [comment]
What about this option for each sitemap? What about using defaults for each sitemap? What about one sitemap whants the `guesser/loader/voter` the other doesn't? Or do you want it as a global flag?

**Comment by ElectricMaxxx** [comment]
solved by #245#
